manifests/kustomize/base/cache-deployer/cluster/cache-deployer-clusterrolebinding.yaml

```diff
@@ -8,5 +8,4 @@ roleRef:
   name: kubeflow-pipelines-cache-deployer-clusterrole
 subjects:
 - kind: ServiceAccount
-  name: kubeflow-pipelines-cache-deployer-sa
-  namespace: kubeflow
\ No newline at end of file
+  name: kubeflow-pipelines-cache-deployer-sa
```

Comment:
You need namespace here.

Reply:
`kustomize` should replace it with the correct namespace:
https://github.com/kubernetes-sigs/kustomize/issues/136 https://github.com/kubernetes-sigs/kustomize/pull/166

Reply:
Thanks for you information! @IronPan @Bobgy